### ISSUE TITLE
files_antivirus app does not pass occ app:check-code

### ISSUE THREAD

**Issue by jnfrmarks** [owncloud/files_antivirus#38]
### Steps to reproduce

1. Install 8.1 enterprise
2. invoke occ app:check-code

### Expected behaviour

The app passes the check

### Actual behaviour

The app fails the check:

```

[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/tests/statustest.php[39m
 1 errors
    [37;41mline   23: OC_App - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/tests/itemtest.php[39m
 3 errors
    [37;41mline   24: OC_User - static method call on private class[39;49m
    [37;41mline   25: OC_User - static method call on private class[39;49m
    [37;41mline   29: OC_User - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/tests/testbase.php[39m
 1 errors
    [37;41mline   25: OC_App - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/tests/scannertest.php[39m
 2 errors
    [37;41mline   45: OC_User - static method call on private class[39;49m
    [37;41mline   49: OC_User - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/tests/bootstrap.php[39m
 1 errors
    [37;41mline   14: OC_Hook - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/templates/notification.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/templates/settings.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/controller/settingscontroller.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/controller/rulecontroller.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/item.php[39m
 2 errors
    [37;41mline  185: OC_DB - static method call on private class[39;49m
    [37;41mline  190: OC_DB - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/notification.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/backgroundscanner.php[39m
 4 errors
    [37;41mline   63: OC_DB - static method call on private class[39;49m
    [37;41mline   81: OC_Util - static method call on private class[39;49m
    [37;41mline   91: OC_Util - static method call on private class[39;49m
    [37;41mline   92: OC_Util - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/scanner.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/status.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/iscannable.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/content.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/activity.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/db/item.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/db/rulemapper.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/db/itemmapper.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/db/rule.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/avirwrapper.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/appconfig.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/scanner/local.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/scanner/external.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/scannerfactory.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/lib/cron/task.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/admin.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/appinfo/app.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/appinfo/routes.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/appinfo/update.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/appinfo/preupdate.php[39m
 3 errors
    [37;41mline    5: OC_DB - static method call on private class[39;49m
    [37;41mline    8: OC_DB - static method call on private class[39;49m
    [37;41mline   13: OC_DB - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/files_antivirus/appinfo/application.php[39m
 0 errors
[37;41mApp is not compliant[39;49m

```

### Server configuration

Ubuntu 14.04
PHP 5.5.9
ownCloud Enterprise Edition 8.1 alpha 1 (daily) Build:2015-04-16T03:08:13+00:00 f50798a674bf223e1c8015b9a27287b2ac6c9f1a

**Comment by DeepDiver1975** [comment]
@VicDeo this should all easily be solvable - THX

**Comment by DeepDiver1975** [comment]
closed as of #39

**Comment by cmonteroluque** [comment]
@DeepDiver1975 @VicDeo this is ready to be closed then?
